### PR TITLE
fixed-Discount-Code-improvement-21214

### PR DESCRIPTION
**Pull request by abrarpathan19**
### Description (*)

Luma theme Apply Discount Code section design improvement according to shopping cart page Discount Coupon box section. please review link : Shopping Cart page : http://prntscr.com/mkvc7o

### Fixed Issues (if relevant)

1. magento/magento2#21214: Luma theme Apply Discount Code section design improvement

### Manual testing scenarios (*)

Checkout page step "Review & Payments" : http://prntscr.com/mkvcoa
It should be according to the shopping cart page coupon box section.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @abrarpathan19. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/design/frontend/Magento/luma/web/css/source/_extends.less [hunk 1]
```diff
@@ -1913,6 +1913,36 @@
             display: table-cell;
         }
     }
+
+    .abs-discount-code {
```
Comment (sivaschenko) [inline app/design/frontend/Magento/luma/web/css/source/_extends.less]: @abrarpathan19 can you please move the fix to Magento_Checkout module of luma theme

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @abrarpathan19 thanks for the pull request, please take a look at the review notes

**Comment by sidolov** [comment]
@abrarpathan19 , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Comment by abrarpathan19** [comment]
@sidolov Thank you :+1:

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-5070](https://jira.corp.magento.com/browse/ENGCOM-5070) has been created to process this Pull Request

**Comment by abrarpathan19** [comment]
@sivaschenko Thanks to approving PR

**Comment by VasylShvorak** [comment]
:heavy_check_mark: QA passed

**Pull request merged**